docs/process/roadmap.md

```diff
@@ -2,22 +2,43 @@
 
 The following are the larger areas of upcoming work the GitHub Desktop team intends to explore. This is not inclusive of everything we're working on, and it's not written in stone. We'll continue to update it as our priorities evolve.
 
-### Onboarding
+#### Onboarding
   
-- Evaluate and improve onboarding for new users: [#5686](https://github.com/desktop/desktop/issues/5686)
-- Measuring success: [#5549](https://github.com/desktop/desktop/issues/5549)
-  
-### Merge conflicts handling
+- Improve onboarding for new users: [#5686](https://github.com/desktop/desktop/issues/5686)
+- Measuring success: [#5549](https://github.com/desktop/desktop/issues/5549) + usability testing
 
-- Improve how Desktop handles merge conflicts: [#5400](https://github.com/desktop/desktop/issues/5400)
-- Measuring success: [#5394](https://github.com/desktop/desktop/issues/5394)
-
-### Support rebase workflows
+#### Support rebase workflows
 
 - Improve rebase workflows: [#5953](https://github.com/desktop/desktop/issues/5953)
 - Measuring success: TBD alongside the work + usability testing
+
+#### Accessibility improvements
+
+- To be further defined, but we want to prioritize scoping a chunk of work on improving accessibility for our users
```

Comment:
Are there something that are clearly in or out of scope? Internationalization, colors used or font used? 

I would imagine internationalization in particular might be highly requested but maybe too large to be part of this effort

Reply:
> Are there something that are clearly in or out of scope? Internationalization, colors used or font used?

The primary focus here is that Desktop is well-supported with screenreaders like NVDA and VoiceOver - that users with accessibility needs are able to navigate and work with Desktop like everyone else.

Reply:
Sounds good. Just thought maybe we could be more specific to help those sorts of users vs color blind users

Reply:
@say25 This one in particular will get more tightly scoped over the next 3 weeks.